doc/source/getting-started/installation.rst

```diff
@@ -391,51 +292,25 @@ please do:
 
 _______________________
 
-.. _installing-qiboicarusq:
-
-qiboicarusq
-^^^^^^^^^^^
-
-The ``qiboicarusq`` package contains the hardware control drivers for chips
-based on superconducting qubits. More details are available at the code
-`repository <https://github.com/qiboteam/qiboicarusq.git>`_.
-
-This backend is designed for laboratories, in order to switch to the ``qiboicarusq``
-backend please do:
-
-.. code-block:: python
-
-      import qibo
-      qibo.set_backend("qiboicarusq")
-
-Installing from source
-""""""""""""""""""""""
-
-The installation procedure presented in this section is useful if you have to
-install and develop the code from source.
-
-In order to install the package perform the following steps:
-
-.. code-block::
-
-      git clone https://github.com/qiboteam/qiboicarusq.git
-      cd qiboicarusq
-
-then proceed with the installation of requirements with:
+.. _docker:
 
-.. code-block::
+Using the code with docker
+--------------------------
 
-      pip install -r requirements.txt
+We provide docker images for tag release of the code using GitHub Packages. The
+docker images contain a pre-configured linux environment with the Qibo
+framework installed with the specific tag version.
 
-Then proceed with the ``qiboicarusq`` installation using ``pip``
+Please refer to the download and authentication instructions from the `Qibo GitHub Packages`_.
 
-.. code-block::
+In order to start the docker image in interactive mode please use docker
+standard syntax, for example:
 
-      pip install .
+.. code::
 
-or if you prefer to manually execute all installation steps:
+    docker run -it ghcr.io/qiboteam/qibo:<tag_version> bash
```

Comment:
I had to use `sudo` in this and all other docker commands but I am not sure if this is relevant here.